docs_samples/01_python_app_insights_overview.md

```diff
@@ -0,0 +1,239 @@
+- Select **Save**
+
+## Configure the environment (Visual Studio Code)
+
+- Open Visual Studio code, select the **Extensions** tab
+- Ensure the following extensions are installed:
+  - ms-python.python
+  - ms-azuretools.vscode-azurefunctions
+
+## Check your Python Version
+
+- In Visual Studio Code, open a Terminal window, run the following command:
+
+```powershell
+python --version
+
+pip install --upgrade pip
+```
+
+- For Python, make sure you see version 3.9.x.  If you do not, do the following:
+  - Download Python and change your path variables to point to the proper version
+  - Restart Visual Studio Code
+- Open the command palette by selecting **View->Command Pallet**
+- Run the `Python: Select interperter` command
+- Select the `Python 3.9.x` version
+
+## Create a Python environment
+
+- Run the following command to create an environment where you can load all the dependencies:
+
+```powershell
+py -3 -m venv .venv
+.venv\scripts\activate
+```
+
+- When prompted, select **yes**
+- Ensure that you select the new environment in the interpertor selection
+
+## Install OpenCensus
+
+[OpenCensus](https://opencensus.io/) is a set of open source libraries hosted on [GitHub](https://github.com/census-instrumentation) that are used to capture trace and metric data from applications.
+
+Metrics can be things like latency, HTTP request and reponse lengths whereas traces are how a request moves through your application layers and other services. Graphs can be built to show how an application or service is performing based on these data points.
+
+To install OpenCensus:
+
+- Open a terminal window, run the following:
+
+```powershell
+pip install --upgrade pip
+
+python -m pip install opencensus
+```
+
+## Install Azure exporter
+
+OpenCensus needs an exporter to know how to send the log and metric data to the target.  Azure provides an exporter for Azure Monitor.
+
+To install the Azure Exporter:
+
+- Open a terminal window, run the following:
+
+```powershell
+python -m pip install opencensus-ext-logging
+python -m pip install opencensus-ext-azure
+python -m pip install opencensus-extension-azure-functions
+python -m pip install opencensus-ext-requests
+python -m pip install pyodbc
+python -m pip install psutil
+
+```
+
+## Configure middleware to track requests
+
+- Browse to the Azure Portal
+- Select your Azure Monitor resource
+- On the **Overview** page, copy the connection string
+- Open the `.env` file and paste it into the `APPLICATIONINSIGHTS_CONNECTION_STRING` value
+
+## Create a simple Python App
+
+- Browse to the Azure Portal.
+- Select your Azure Monitor resource.
+- On the **Overview** page, copy the connection string and save it for later.
+- Create a new python file called `app.py`.
+- Copy the following into it, be sure to replace your Application Insights connection string:
+
+```python
+import logging
+
+from opencensus.ext.azure.log_exporter import AzureLogHandler
```

Comment:
What is the purpose of the document? This sample uses an example that is already existing in [examples](https://github.com/census-instrumentation/opencensus-python/blob/master/contrib/opencensus-ext-azure/examples/logs/simple.py).